meetup-2023-10-19: add announcement draft (#5)
nix-community-bochum/nix-learning-group: meetups/19-10-2023/announcement.md

Reviewer: Please use a sane time format: `2023-10-19` :laughing:

## Announcement text draft

### Was ist Nix?

> Nix ist ein Package Manager, mit dem Software deklarativ, hermetisch und reproduzierbar gebaut werden kann. Die Software-Pakete werden dafür in der funktionalen Programmiersprache Nix beschrieben. NixOS ist eine Linux-Distribution, die auf dem Nix-Paketmanager und der Nix-Programmiersprache aufbaut. Das ganze System wird deklarativ in einer Config beschrieben und lässt sich (u.A.) einfach reproduzieren und in Git verwalten. Dabei bricht Nix mit etablierten Linux-Konzepten und eignet sich ideal, um sein Verständnis für Linux und Software generell zu vertiefen.

Reviewer: ```suggestion
Nix ist ein Package Manager, mit dem du Software deklarativ, hermetisch und reproduzierbar bauen kannst. Die Software-Pakete werden dafür in der funktionalen Programmiersprache Nix beschrieben. NixOS ist eine Linux-Distribution, die auf Nix aufbaut. Das ganze System wird deklarativ in einer Config beschrieben und lässt sich u.A. einfach reproduzieren und in Git verwalten. Dabei bricht Nix mit etablierten Linux-Konzepten und eignet sich ideal, um sein Verständnis für Linux und Software generell zu vertiefen.
```
I think we should address people directly here, maybe even more than in the initial version.

Mehr Informationen zu Nix/NixOS gibt es auf der [offiziellen Website](https://nixos.org/).

### Was ist die Learning Group?

In regelmäßigen Treffen wollen wir mit euch zusammen unser Wissen über Nix(OS) vertiefen, zusammen an Systemkonfigurationen hacken und Software packagen. Explizit wollen wir auch Menschen ansprechen, die Nix noch nicht kennen, aber gerne ausprobieren würden und euch beim Einstig unterstützen.

> ### Was muss ich mitbringen?

Reviewer: ```suggestion
### Was solltest du mitbringen?
```
I think I liked the previous shorter headlines better.

> Bring am besten einen Laptop mit Linux mit. Alternativ könnt ihr auch einen bootbaren USB-Stick mitbringen, oder euch eine VM aufsetzen. Wir helfen euch natürlich auch gern, Nix/NixOS auf jedem System, das ihr mitbringt, zu installieren.

Reviewer: ```suggestion
Bring am besten einen Laptop mit Linux mit. Alternativ könnt ihr auch einen bootbaren USB-Stick mitbringen, oder euch eine VM aufsetzen. Wir helfen euch natürlich auch gern, Nix/NixOS auf einem System, das ihr mitbringt, zu installieren.
```

### Pizza?

> Pizza! Und Getränke. Sag uns vorher am besten Bescheid, dass du planst zu kommen, damit wir entsprechend kalkulieren können.

Reviewer: ```suggestion
Pizza! Und Getränke. Sag uns vorher am besten Bescheid, dass du planst zu kommen.
```
Imo obvious.
